diff --git a/include/linux/swap.h b/include/linux/swap.h
index 4cc6ec3..48d5a8e 100644
--- a/include/linux/swap.h
+++ b/include/linux/swap.h
@@ -365,6 +365,8 @@ extern unsigned long zone_reclaimable_pages(struct zone *zone);
 extern void lru_add_drain_cpu(int cpu);
 extern void lru_add_drain_cpu_zone(struct zone *zone);
 extern void lru_add_drain_all(void);
+extern int request_reclaim_flusher_wakeup(void);
+extern unsigned long reclaim_page(struct page *page);
 extern void rotate_reclaimable_page(struct page *page);
 extern void deactivate_file_page(struct page *page);
 extern void deactivate_page(struct page *page);
diff --git a/mm/vmscan.c b/mm/vmscan.c
index 562e87c..04674e9 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -3646,6 +3646,7 @@ static int balance_pgdat(pg_data_t *pgdat, int order, int highest_zoneidx)
 	return sc.order;
 }
 
+//__reclaim_page.merge//
 /*
  * The pgdat->kswapd_highest_zoneidx is used to pass the highest zone index to
  * be reclaimed by kswapd from the waker. If the value is MAX_NR_ZONES which is
