diff --git a/include/linux/swap.h b/include/linux/swap.h
index f3e0ac20c2e8..62ff44290676 100644
--- a/include/linux/swap.h
+++ b/include/linux/swap.h
@@ -382,6 +382,8 @@ void folio_add_lru_vma(struct folio *, struct vm_area_struct *);
 void mark_page_accessed(struct page *);
 void folio_mark_accessed(struct folio *);
 
+extern int request_reclaim_flusher_wakeup(void);
+extern unsigned long reclaim_page(struct folio *folio);
 extern atomic_t lru_disable_count;
 
 static inline bool lru_cache_disabled(void)
diff --git a/mm/vmscan.c b/mm/vmscan.c
index 28ba2b06fc7d..89f9f714f1a8 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -2150,6 +2150,7 @@ static unsigned int reclaim_folio_list(struct list_head *folio_list,
 	return nr_reclaimed;
 }
 
+//__reclaim_page.merge//
 unsigned long reclaim_pages(struct list_head *folio_list)
 {
 	int nid;
