diff --git a/include/linux/swap.h b/include/linux/swap.h
index 0ceed49516ad..e1c1da34dc9a 100644
--- a/include/linux/swap.h
+++ b/include/linux/swap.h
@@ -384,6 +384,8 @@ void folio_add_lru_vma(struct folio *, struct vm_area_struct *);
 void mark_page_accessed(struct page *);
 void folio_mark_accessed(struct folio *);
 
+extern int request_reclaim_flusher_wakeup(void);
+extern unsigned long reclaim_page(struct page *page);
 extern atomic_t lru_disable_count;
 
 static inline bool lru_cache_disabled(void)
diff --git a/mm/vmscan.c b/mm/vmscan.c
index 5b7b8d4f5297..42df69802f35 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -2724,6 +2724,7 @@ static unsigned int reclaim_folio_list(struct list_head *folio_list,
 	return nr_reclaimed;
 }
 
+//__reclaim_page.merge//
 unsigned long reclaim_pages(struct list_head *folio_list)
 {
 	int nid;
